member_registry.md

```diff
@@ -1,15 +1,15 @@
 # Cooperative Membership
 | Keybase Name | Ethereum Address |
 |----------|:-------------:|
-| <a target="_blank" href="https://keybase.io/lkngtn">@lkngtn</a> |  `0x625236038836CecC532664915BD0399647E7826b` |
-| <a target="_blank" href="https://keybase.io/stellarmagnet">@stellarmagnet</a> | `0xeBAC3108C58Bd4dD48a8110B7750480Cea7bC3Fb`  |
-| <a target="_blank" href="https://keybase.io/jjperezaguinaga">@jjperezaguinaga</a> | `0x016b4De659f33f27e0C9281E41C5EBd6942c6f61` |
-| <a target="_blank" href="https://keybase.io/bingen">@bingen</a> | `0x6F82dfD56A399390EFe5140d8E0390B200Dbb935` |
-| <a target="_blank" href="https://keybase.io/imestin">@imestin</a>  | `0xF971b59792c8D122602b37E00d310fE069A03619` |
-| <a target="_blank" href="https://keybase.io/louisgrx">@louisgrx</a> | `0xbe40DCADB6bF4De1c7810EcA3504E5f3BEA9cF35` |
-| <a target="_blank" href="https://keybase.io/maxsemenchuk">@maxsemenchuk</a> | `0xA1c4753Ba0abb8230765660c662CCdE5B801Fb54` |
-| <a target="_blank" href="https://keybase.io/rperez89">@rperez89</a>  | `0x593e1F9809658d0c92e9f092cF01Aad7D0d734f3` |
-| <a target="_blank" href="https://keybase.io/ro5s">@ro5s</a> | `0x1409f15D77F28882C220860d5F534A1A7a9F3481` |
-| <a target="_blank" href="https://keybase.io/schmidsi">@schmidsi</a> | `0xe98C357458C89198575a32462760402CaB2A9a4D` |
-| <a target="_blank" href="https://keybase.io/goliat">@goliat</a> | `0x3E0B02c9f2c5107c2747fCb0E68d997bE2fa0F7f` 
+| [@lkngtn](https://keybase.io/lkngtn) |  `0x625236038836CecC532664915BD0399647E7826b` |
+| [@stellarmagnet](https://keybase.io/stellarmagnet) | `0xeBAC3108C58Bd4dD48a8110B7750480Cea7bC3Fb`  |
+| [@jjperezaguinaga](https://keybase.io/jjperezaguinaga) | `0x016b4De659f33f27e0C9281E41C5EBd6942c6f61` |
+| [@bingen](https://keybase.io/bingen) | `0x6F82dfD56A399390EFe5140d8E0390B200Dbb935` |
+| [@imestin](https://keybase.io/imestin)  | `0xF971b59792c8D122602b37E00d310fE069A03619` |
+| [@louisgrx](https://keybase.io/louisgrx) | `0xbe40DCADB6bF4De1c7810EcA3504E5f3BEA9cF35` |
+| [@maxsemenchuk](https://keybase.io/maxsemenchuk) | `0xA1c4753Ba0abb8230765660c662CCdE5B801Fb54` |
+| [@rperez89](https://keybase.io/rperez89)  | `0x593e1F9809658d0c92e9f092cF01Aad7D0d734f3` |
+| [@ro5s](https://keybase.io/ro5s) | `0x1409f15D77F28882C220860d5F534A1A7a9F3481` |
+| [@schmidsi](https://keybase.io/schmidsi) | `0xe98C357458C89198575a32462760402CaB2A9a4D` |
+| [@goliat](https://keybase.io/goliat) | `0x3E0B02c9f2c5107c2747fCb0E68d997bE2fa0F7f` 
```

Comment:
Missing a pipe at the end

Reply:
I guess that may not have been your fault - do i just fix it? :)

Reply:
No worries, I went ahead and fixed it ☺️